build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -331,3 +331,25 @@ multicluster:
 # The default is antrea-agent's Namespace.
   namespace: {{ .namespace | quote }}
 {{- end }}
+
+secondaryNetworkOVS:
```

Comment:
should we have a more general `secondaryNetwork` section and a subsection for OVS configuration? cc @jianjuns 

applies to the values.yaml file as well

Reply:
I feel we should rename "secondaryNetworkOVS" to `secondaryNetwork`.

No strong opinion, we should have an "ovs" subsection (for ovsIntegrationBridgeName, ovsTransportBridgeName, ovsDatapathType, and ovsPatchPort"; tunnelType should be separate. Either way works for me.